plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-products-controller.php

```diff
@@ -1220,6 +1220,12 @@ public function get_item_schema() {
 						),
 					),
 				),
+				'has_options'     => array(
+					'description' => __( 'Shows if the product needs confiruging before it can be bought.', 'woocommerce' ),
```

Comment:
Typo ('confiruging'). Perhaps:

```suggestion
					'description' => __( 'Shows if the product needs to be configured before it can be bought.', 'woocommerce' ),
```

Reply:
(Went ahead and updated this since it was a minor thing.)